web/client/components/share/SharePanel.jsx

```diff
@@ -152,6 +152,13 @@ class SharePanel extends React.Component {
             !isEqual(this.props?.viewerOptions?.orientation, newProps.viewerOptions?.orientation)) {
```

Comment:
I think you need to revisit this condition and below code changes.

Maybe you can remove this condition and skip the state update of orientation fields from [here](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/share/SharePanel.jsx#L195-L197), as you are doing it explicitly below